ui-components/src/blocks/Card.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import { Heading } from "../headers/Heading"
+import "./Card.scss"
+
+export interface CardHeaderProps {
+  /** An additional element(s) you can add to the side of the main child element(s) */
+  suffix?: React.ReactNode
+  /** Additional class name */
+  className?: string
+  children: React.ReactNode
+}
+
+const CardHeader = (props: CardHeaderProps) => {
+  const classNames = ["card__header"]
+  if (props.suffix) classNames.push("is-flex")
+  if (props.className) classNames.push(props.className)
+
+  return (
+    <header className={classNames.join(" ")}>
+      {props.children}
+      {props.suffix}
+    </header>
+  )
+}
+
+export interface CardHeadingGroupProps {
+  /** A string or element to display in an `h2` tag (overridable via `headingPriority`) */
+  heading: React.ReactNode
+  /** A string or element to display in an `p` tag (using `aria-roledescription="subtitle"`) */
+  subheading: React.ReactNode
+  /**
+   * The heading level (1 through 6)
+   * @default 2
+   */
+  headingPriority?: number
+  /** Additional class name for the heading */
+  headingClassName?: string
+  /** Additional class name for the subheading */
+  subheadingClassName?: string
+}
+
+const CardHeadingGroup = (props: CardHeadingGroupProps) => {
+  return (
+    <hgroup role="group">
+      <Heading className={props.headingClassName} priority={props.headingPriority ?? 2}>
+        {props.heading}
+      </Heading>
+      <p className={props.subheadingClassName} aria-roledescription="subtitle">
+        {props.subheading}
+      </p>
+    </hgroup>
+  )
```

Comment:
Should the Heading Group live outside of the Card Component? I thought the intent of #2936 was to make the headerGroup as a standalone component

Reply:
Oh, hmm that's a good point. I could extract that out to a separate file/className.